ui/apps/platform/src/Containers/VulnMgmt/widgets/MostCommonVulnerabilities.js

```diff
@@ -53,16 +71,22 @@ const processData = (data, workflowState) => {
 };
 
 const MostCommonVulnerabilities = ({ entityContext, search, limit }) => {
+    const { isFeatureFlagEnabled } = useFeatureFlags();
+    const showVMUpdates = isFeatureFlagEnabled('ROX_FRONTEND_VM_UDPATES');
+
     const entityContextObject = queryService.entityContextToQueryObject(entityContext); // deals with BE inconsistency
 
     const queryObject = { ...entityContextObject, ...search }; // Combine entity context and search
     const query = queryService.objectToWhereClause(queryObject); // get final gql query string
+    const GQL_QUERY = showVMUpdates
```

Comment:
nit: I think ALL_CAPS should be reserved for true constants.

Not worth changing, since this code should only live for a couple of months, unless there are other changes needed.

```suggestion
    const queryToUse = showVMUpdates
```